common/app/conf/switches.scala

```diff
@@ -536,6 +545,16 @@ object Switches {
     safeState = Off, sellByDate = new LocalDate(2015, 4, 23)
   )
 
+  val ABHeadlineSwitches = (1 to 10) map { n =>
```

Comment:
HMMM ... they're 10 different AB tests, so I think so, yes, as it looks up the switch by the AB test ID.